documentation/src/main/wiki/GettingStarted.md

```diff
@@ -26,7 +26,7 @@ First download Knot.x sample app & config for latest version, or build it yourse
 
 Now you can run Knot.x:
 ```
-java -Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.SLF4JLogDelegateFactory -Dlogback.configurationFile=config/logback.xml -jar knotx-example-app-X.Y.Z-fat.jar -conf config/bootstrap.json
+java -Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.SLF4JLogDelegateFactory -cp config:target/knotx-example-app-X.Y.Z-fat.jar io.vertx.core.Launcher
```

Comment:
Shouldn't be here `... -cp config:lib/* io.vertx.core.Launcher` ?
And I believe few lines before we should define to download `knotx-example-app-X.Y.Z.fat.jar` into `lib` directory.